models/enrollment_journey_model.sql

```diff
@@ -0,0 +1,42 @@
+{{ config(
+  materialized='table'
+) }}
+
+WITH enrollment_data AS (
+    SELECT
+        er.academic_year,
+        er.is_current,
+        er.start_date AS start_date,
```

Comment:
Why are these 4 lines (9-13) needed in terms of the `AS` clause?

Reply:
Removed.